src/test/java/com/hotjava/app/hotjava/HotJavaApplicationTests.java

```diff
@@ -1,234 +0,0 @@
-package com.hotjava.app.hotjava;
-
-import com.hotjava.app.hotjava.dao.IVehicleDAO;
-import com.hotjava.app.hotjava.dto.Photo;
-import com.hotjava.app.hotjava.dto.Vehicle;
-import com.hotjava.app.hotjava.service.IVehicleService;
-import com.hotjava.app.hotjava.service.VehicleService;
-import org.junit.jupiter.api.Test;
-import org.mockito.Mockito;
-import org.springframework.beans.factory.annotation.Autowired;
-import org.springframework.boot.test.context.SpringBootTest;
-import org.springframework.boot.test.mock.mockito.MockBean;
-
-import java.util.List;
-
-import static org.junit.jupiter.api.Assertions.*;
-import static org.mockito.Mockito.*;
-
-@SpringBootTest
-class HotJavaApplicationTests {
-
-	private IVehicleService vehicleService;
-	private Vehicle mockVehicle = new Vehicle();
-
-	@MockBean
-	private IVehicleDAO vehicleDAO;
```

Comment:
Since the IVehicleDAO is mocked both the 'vehicleService', and vehicleServiceNoMock are using it. This is what caused the Test ''verifyServiceCanAddVehicles" to fail.